Makefile

```diff
@@ -4,6 +4,21 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 .ONESHELL .PHONY: up update-box destroy-box remove-tmp clean example
 .DEFAULT_GOAL := up
 
+###################################
+########## Color Scheme ###########
+###################################
+BLACK        := \033[0;30m
+RED          := \033[0;31m
+GREEN        := \033[0;32m
+ORANGE       := \033[0;33m
+BLUE         := \033[0;34m
+PURPLE       := \033[0;35m
+CYAN         := \033[0;36m
+LIGHTGREY    := \033[0;37m
+DARKGREY     := \033[1;30m
+YELLOW       := \033[1;33m
+WHITE        := \033[1;37m
+RESET        := \033[0m
```

Comment:
Do these colors also work for Linux terminal?

Reply:
Yes I believe so! Both linux and mac run bash/zsh which both can interpret ANSI-codes. -@pdmthorsrud